index.js

```diff
@@ -1,24 +1,53 @@
-import './src/js/media-play-button.js';
-import './src/js/media-forward-button.js';
-import './src/js/media-replay-button.js';
-
-import './src/js/media-thumbnail-preview-element.js';
-import './src/js/media-progress-range.js';
-
-import './src/js/media-mute-button.js';
-import './src/js/media-volume-range.js';
-
-
-import './src/js/media-current-time-display.js';
-import './src/js/media-duration-display.js';
-
-import './src/js/media-playback-rate-button.js';
-import './src/js/media-fullscreen-button.js';
-import './src/js/media-pip-button.js';
-
-import './src/js/media-title-element.js';
-
-import './src/js/media-control-bar.js';
-
-// Main <media-chrome> element
-import './src/js/media-chrome-container.js';
+import MediaChromeButton from './src/js/media-chrome-button.js';
+import MediaChromeContainer from './src/js/media-chrome-container.js';
```

Comment:
I changed the class name of MediaChrome to MediaChromeContainer to disambiguate it from the main project, however I left the tag name itself `<media-chrome>`. However with this "load each element" approach, it might throw people off who are trying to load the media-chrome tag. Interested in your thoughts.

Reply:
I think it'd be a bit confusing to not have something match the name. Would it be crazy to just have `<media-chrome>` as an alias for `<media-chrome-container>`?

Reply:
I tried that and it doesn't let you use the same class for two different tag names. A hack might be to subclass the class and use that to register a second tag name.

I like the discoverability that using <media-chrome> as a tag name adds, but think that could also be confusing with the parent project. Should it just be `@media-chrome/media-container`? Probably overthinking this.

Either way, I think this is good to merge without solving this issue.

Reply:
I think `media-container` feels pretty good imo.

Reply:
Yeah, cool. Let's do it. It's a breaking change, but we can do the subclass hack while we deprecate the `<media-chrome>` version. But not needed in this PR.